docs/source/contributing.md

```diff
@@ -0,0 +1,154 @@
+
+The most up-to-date 'source of truth' will be the specification on the `main` branch of the specification repository.
+The community can decide when to 'tag' a new version of the specification.
+They may also decide to where to publish the specification.
+
+The specification repository is self-contained and relates only to the specification specifically, or its governance.
+Any contributions to the wider SATRE project should be made through a different medium via the [SATRE GitHub organisation](https://github.com/sa-tre), or by contacting the SATRE team at [satre-contact@dundee.ac.uk](mailto:satre-contact@dundee.ac.uk).
+
+### Contribution Process
+
+Issues should be used to discuss ideas, potential changes and to ask questions.
+Issue templates have been designed for common issue archetypes to help collect the most important information and present it in a clear, consistent way.
+It is possible, however, to open a blank issue if none of the templates are suitable.
+
+When ready, changes will be proposed in pull requests.
+Similarly to issues, there is a pull request template.
+This template prompts contributors to include important details which helps explain the contribution and makes triage and review easier.
+
+Pull requests will be used to review changes.
+During the review process, the pull request will be used for discussion, to suggest amendments and ultimately accept or reject the change.
+
+We use this process to ensure that as much as possible of the discussion and decision making process can be public.
+This is to provide as open and accessible as possible an environment for all contributors to engage in the conversation.
+
+### Consensus Mechanism
+
+Approval from the SATRE team is based on [lazy consensus](https://medlabboulder.gitlab.io/democraticmediums/mediums/lazy_consensus/).
+Once a pull request has been open for 7 days, unless there are outstanding objections within the comments, the SATRE team will approve the change.
+
+If any of the SATRE team objects to a particular issue, this will be raised by individuals in the comments of the pull request.
+The SATRE team will only approve pull requests that have no outstanding objections.
+
+## Writing in Markdown
+
+The [Myst Parser](inv:myst-parser:std:doc#syntax/typography) documentation has a guide on the Markdown format used in the specification source files.
+GitHub also has a helpful page on [getting started with writing and formatting on GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github), which will be useful when writing Markdown for GitHub (for example in issue or pull request comments).
+
+You can think of Markdown as a few little symbols around your text that instruct how to render the text.
+For example, you could write words in **bold** (`**bold**`), in _italics_ (`_italics_`), or as a [link](https://youtu.be/dQw4w9WgXcQ) (`[link](https://youtu.be/dQw4w9WgXcQ)`) to another web page.
```

Comment:
🤔

Reply:
I assume that is not approval 😄. Any suggestions?

This is mostly lifted from the draft in #5. I made some formatting changes and added the note above about the Myst Markdown syntax (as most of the content should use that rather than GitHub Markdown).

I'm not particularly attached to this. Not sure if the link sets the right tone 🙂.I think we need something to explain what Markdown is (and why we use it?). I think we should be careful not to try and write an introduction to Markdown or a Markdown guide.